modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/Loading.spec.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {render} from '@testing-library/react';
+import React from 'react';
+
+import Loading from './Loading';
+
+describe('Loading', () => {
+	it('renders Loading component without crashing', () => {
+		const {container} = render(<Loading />);
+
+		const loadingElement = container.querySelector(
+			'span.loading-animation'
+		);
```

Comment:
You can get loading from 
```suggestion
		const loadingElement = screen.getByTestId('loading')
```

Reply:
@interaminense done, but I kept this selector because `span.loading-animation` is a child of `screen.getByTestId('loading')`. And I needed the first one to make some tests more accurate. Please check it what I've added and see if makes sense. Thanks!